test/osw_files/SFD_2000sqft_2story_SL_FA_Denver_Windows_Skylights.osw

```diff
@@ -68,7 +68,7 @@
         "overhang_left_facade": "true",
         "overhang_offset": "0.5",
         "overhang_right_facade": "true",
-        "right_skylight_area": "10",
+        "right_skylight_area": "0",
```

Comment:
Instead of changing this to 0, shoudn't `left_skylight_area` be changed to 10?

Reply:
The model only has front/back roofs, so only front/back skylight areas can be non-zero. The original was producing an error that skylight area > 0 but there is no roof to put it on, that's what this change addresses.